string name  # for descriptive purposes
string type  # opaque or transparent 
float32[3] color  # RGB for visualization node
float32 min_return  # 0.0 to 1.0 representing the minimum chance of return
float32 max_return  # 0.0 to 1.0 representing the maximum chance of return
float32 angular_return  # the chance of return lost per degree from perpendicular
